index.html

```diff
@@ -223,27 +223,27 @@ <h3>Vincent Villafuerte</h3><br><span>Founder</span>
 
                 <div class="team__card">
                     <div class="team__content">
-                        <div class="team__img"><img src="./assets/img/owl.png" alt=""></div>
+                        <div class="team__img"><img src="./assets/img/tian.jpg" alt="tian"></div>
                         <div class="team__text">
-                            <h3>someones name</h3><br><span>Moderator</span>
+                            <h3>Christian Igbaras</h3><br><span>Moderator</span>
```

Comment:
Not ~~Moderator~~, put your role as **Administrator**

Reply:
Ok noted!